Configuration/TCA/tx_beautyofcode_domain_model_flexform.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
Does this work probably? Not quite sure because after all, its not a real table.